python/pyspark/pandas/plot/core.py

```diff
@@ -437,6 +438,37 @@ def get_fliers(colname, outliers, min_val):
 
         return fliers
 
+    @staticmethod
+    def get_multicol_fliers(colnames, multicol_outliers, multicol_whiskers):
+        scols = []
+        extract_colnames = []
+        for i, colname in enumerate(colnames):
+            formated_colname = "`{}`".format(colname)
+            outlier_colname = "__{}_outlier".format(colname)
+            min_val = multicol_whiskers[colname]["min"]
```

Comment:
I feel it weird to select the outliers by the distance `|value - lower_whisker|`, which is used in series.boxplot.

It should be something like `|value - median|` or `|value - mean|`, will revisit this later.